src/pages/Home.tsx

```diff
@@ -1,46 +1,76 @@
+import { defaultOutline, semiTransparentBackground } from 'context/Theme';
 import usePageTitle from 'hooks/usePageTitle';
-import { useMemo } from 'react';
+import { Plus, Xmark } from 'iconoir-react';
 import { FormattedMessage } from 'react-intl';
 
 const Home = () => {
-    usePageTitle({ header: authenticatedRoutes.home.title });
-
-    const homePageError = useGlobalSearchParams(
-        GlobalSearchParams.HOME_PAGE_ERROR
-    );
+    const theme = useTheme();
 
-    // home page error related
-    const notificationMessage = useMemo(() => {
-        if (homePageError) {
-            return 'directives.grant.skipped.message';
-        }
+    usePageTitle({ header: authenticatedRoutes.home.title });
 
-        return null;
-    }, [homePageError]);
+    const { welcomeShown, setWelcomeShown } = useShowDashboardWelcome();
 
     return (
         <>
-            {notificationMessage ? (
-                <LoginNotifications
-                    notificationMessage={notificationMessage}
-                    notificationTitle="directives.grant.skipped.title"
-                />
-            ) : null}
-            <Toolbar sx={{ justifyContent: 'center', mb: 3 }}>
-                <Typography variant="h4">
-                    <FormattedMessage id="home.main.header" />
-                </Typography>
-            </Toolbar>
-
-            <HeroImageAndDescription />
+            <LoginError />
+
+            <Box
+                sx={{
+                    background: semiTransparentBackground[theme.palette.mode],
+                    border: defaultOutline[theme.palette.mode],
+                    borderRadius: 3,
+                }}
+            >
+                <Toolbar
+                    disableGutters
+                    sx={{
+                        justifyContent: welcomeShown ? 'center' : 'flex-start',
+                        px: 2,
+                    }}
+                >
+                    <Typography
+                        variant="h5"
+                        sx={{ fontSize: welcomeShown ? 24 : 18 }}
+                    >
+                        <FormattedMessage id="home.main.header" />
+                    </Typography>
+
+                    <IconButton
+                        onClick={() => setWelcomeShown(!welcomeShown)}
+                        style={{ position: 'absolute', top: 14, right: 8 }}
+                    >
+                        {welcomeShown ? (
+                            <Xmark
+                                style={{ color: theme.palette.text.primary }}
+                            />
+                        ) : (
+                            <Plus
+                                style={{ color: theme.palette.text.primary }}
+                            />
+                        )}
+                    </IconButton>
+                </Toolbar>
+
+                <Collapse in={welcomeShown}>
+                    <HeroImageAndDescription />
+                </Collapse>
+            </Box>
```

Comment:
Future work - should make this also a stand alone component since it is the only thing in here that cares about the `welcomeShown` flag.